README.md

```diff
@@ -76,50 +84,45 @@ To use Device Simulator Express, install the extension from the marketplace and
 
 #### 1. Start with the "Device Simulator Express [Circuit Playground Express]: New File" Command.
 
-1. Type in `"Device Simulator Express: [Circuit Playground Express] New File"` in the command palette (`CTRL+SHIFT+P` to open the command palette).  
-   <img alt='"New File" animation' src=https://raw.githubusercontent.com/microsoft/vscode-python-devicesimulator/dev/assets/readmeFiles/newFile.gif>
-2. Name and save your file somewhere, and we’re good to go! (3)
+1. Type in `"Device Simulator Express: [Circuit Playground Express] New File"` in the command palette (`CTRL/CMD + SHIFT + P` to open the command palette).  
+   <img alt='"New File" animation' src='assets/readmeFiles/cpx/cpx-new-file.gif'>
+2. Name and save your file somewhere, and we’re good to go!
 3. Start with some examples: you can find examples files and tutorials inside the comments, as well as in the notification pop up when you run the `"Device Simulator Express: [Circuit Playground Express] New File"` Command.
 
-<img alt='How to find example code screenshot' src=https://raw.githubusercontent.com/microsoft/vscode-python-devicesimulator/dev/assets/readmeFiles/findExamples.jpg>
-
 #### 2. Start from an existing python file.
 
 1. Open the folder or your .py file in Visual Studio Code.
 2. Run `Device Simulator Express: [Circuit Playground Express] Open Simulator` from the command palette or icon in the editor toolbar.
 
 #### 3. Run your code on the simulator.
 
-<img alt='How to run the simulator animation' src='https://github.com/microsoft/vscode-python-devicesimulator/blob/dev/assets/readmeFiles/run.gif?raw=true'>
+<img alt='How to run the simulator animation' src='assets/readmeFiles/cpx/cpx-run.gif'>

```

Comment:
If it's not too much work, I would have the pop up show from the extension instead from `__DISPLAYNAME__ (Extension)`.

Reply:
Sorry, not sure what you mean by this

Reply:
My bad! If you take a look at this gif, when you you try to run the simulator a pop up shows up that asks if you agree to run python code on your machine. The extension that sends the pop up is `__DISPLAYNAME__ (Extension)` instead of `Device Simulator Python` (or whatever the official name was).
If it's not too much work to re-record the gif, I think it would be more clearer to the readers of the file when they get the same pop up from the same extension.
Let me know if it's still not clear 😄

Reply:
Fixed